tox.ini

```diff
@@ -44,47 +44,48 @@ usedevelop = true
 deps = pytest
 
 [testenv:py310]
-extras = tests
+dependency_groups = tests
 commands =
     pytest --cov buku -vv -m "not non_tox" {posargs}
 
 [testenv:py311]
-extras = tests
+dependency_groups = tests
 commands =
     pytest --cov buku -vv -m "not non_tox" {posargs}
 
 [testenv:py312]
-extras = tests
+dependency_groups = tests
 commands =
     pytest --cov buku -vv -m "not non_tox" {posargs}
 
 [testenv:py313]
-extras = tests
+dependency_groups = tests
 commands =
     pytest --cov buku -vv -m "not non_tox" {posargs}
 
 [testenv:py314]
-extras = tests
+dependency_groups = tests
 commands =
     pytest --cov buku -vv -m "not non_tox" {posargs}
 
 [testenv:quick]
 basepython = {env:BASEPYTHON:py313}
-extras = tests
+dependency_groups = tests
 commands =
     pytest --cov buku -vv -m "not non_tox and not slow" {posargs}
 
 [testenv:nogui]
 basepython = {env:BASEPYTHON:py313}
-extras = tests
+dependency_groups = tests
 commands =
     pytest --cov buku -vv -m "not non_tox and not gui" {posargs}
 
 [testenv:pylint]
 basepython = {env:BASEPYTHON:py313}
 deps =
     pylint
-    .[tests]
+    .
+dependency_groups = tests
```

Comment:
…Come to think of it, `pylint` is already included in the `tests` group; there's no need to mention it separately. And the same applies to `pytest` & `flake8`. (On the other hand, the tests from this config are failing when run locally :sweat_smile:)

You'll want to remove all `deps` & `dependency_groups` (in all sections), and place this in the `[testenv]` section instead:
```ini
deps = .
dependency_groups = tests
```
(`deps = .` is apparently not optional)

Reply:
> (`deps = .` is apparently not optional)

…Might want to double-check that after fixing your `pyproject.toml` so that it actually works correctly :sweat_smile:

Reply:
> Come to think of it, pylint is already included in the tests group

Totally missed that, removed

> Might want to double-check

Looks like this is a current limitation of tox: https://github.com/tox-dev/tox/issues/3561. It installs dep groups first and without the project, i.e. `pip install ... buku[server,locales] ...`, which pip of course interprets as the remote PyPI `buku` package and not our local one (unlike `pip install . --group tests`). I just specified those extras manually instead in `tox.ini` for now